openedx/core/djangoapps/theming/helpers.py

```diff
@@ -21,6 +21,14 @@ def get_template_path(relative_path, **kwargs):
     """
     This is a proxy function to hide microsite_configuration behind comprehensive theming.
     """
+    # Import is placed here to avoid circular import
+    from openedx.core.djangoapps.theming.microsite_transition_helpers import current_request_has_associated_site_theme
+
+    # We need to give priority to theming over microsites
+    # So, it there is a theme then we disable microsite.
```

Comment:
We're not really disabling the microsite here, we're simply shortcutting the workflow and returning the provided relative_path